### PR TITLE
DB setup and working pulling data from the db

### PR DESCRIPTION
**Pull request by AleksandraZhi**
1) Setup a working connection to a remote mongo db
2) Website is now able to pull information from the db onto the page, displaying it nicely a page.
Attached is the way a pulled info from a db looks like (novel's page, novel "Rockstar")

(https://user-images.githubusercontent.com/42120918/106050171-dc4d6d00-60de-11eb-8cfc-9830bfa00aa5.JPG)

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const mongoose = require("mongoose");
 const routes = require("./routes/index");
 const helpers = require("./helpers");
 const path = require('path');
+const year = require('year');
```
Comment (mforner13) [inline server/app.js]: It looks like you don't need this import here as it's not used in this file :)

File: server/controllers/mainController.js [hunk 2]
```diff
@@ -1,3 +1,24 @@
-exports.novelsPage = (req, res) => {
-  res.render("../views/main");
+
+const mongoose = require('mongoose');
+const Novel = mongoose.model('Novel');
+const path = require('path');
+const fs = require('fs');
```
Comment (mforner13) [inline server/controllers/mainController.js]: Is this used in this file? :)

File: server/models/Books.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+mongoose.Promise = global.Promise;
+
+const Schema = mongoose.Schema;
+
+const novelSchema = new mongoose.Schema ({
+  bookType: String,  //types of books: novel, shortStory, inspiredStory, article
+  bookTitle: String,
+  bookChapter: String,  //can be chapter name or chapter number
```
Comment (mforner13) [inline server/models/Books.js:8]: How will you distinguish between the novels which have chapter names or chapter numbers?

File: server/models/Books.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+mongoose.Promise = global.Promise;
+
+const Schema = mongoose.Schema;
+
+const novelSchema = new mongoose.Schema ({
```
Comment (mforner13) [inline server/models/Books.js:6]: If this model will be storing more than novels, I would use a more generic name name like Story? :) 

Alternatively, if there's a difference between what you would store against each type of story, I think having separate models would be cleaner.

File: server/models/Books.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+//   type: String,
+//   title: String,
+//   text: String,
+//   slug: String,
+// })
+
+module.exports = mongoose.model("Novel", novelSchema);
+// module.exports = mongoose.model("Novel", novelSchema);
+// module.exports = mongoose.model("Short", shortsSchema);
+// module.exports = mongoose.model("Article", storiesArticlesSchema);
```
Comment (mforner13) [inline server/models/Books.js]: It's good practice to not commit commented out code, as it is confusing for others reading it why it's there, and also makes the file more difficult to understand. Are you planning to use these models further down the line? :)

File: server/views/novels-main.pug [hunk 6]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (mforner13) [inline server/views/novels-main.pug]: Looks like some stuff left in from a merge conflict :) And one at the bottom of the file too

**Review by mforner13 — COMMENTED** [review]
Looks really good! 

Just a couple of comments 😄

File: server/app.js (hunk 1 above)
Comment (AleksandraZhi) [reply server/app.js]: removed

File: server/controllers/mainController.js (hunk 2 above)
Comment (AleksandraZhi) [reply server/controllers/mainController.js]: I'm useing model and mongoose to manipulate models. The rest is removed now.

File: server/models/Books.js (hunk 3 above)
Comment (AleksandraZhi) [reply server/models/Books.js:8]: novel schema deals only with novels as it has chapters field. I will create a separate schema for the other categories.

File: server/models/Books.js (hunk 4 above)
Comment (AleksandraZhi) [reply server/models/Books.js:6]: Novels does have a chapter field. The other schemas won't have this.

File: server/models/Books.js (hunk 5 above)
Comment (AleksandraZhi) [reply server/models/Books.js]: Cleaned all commented out code.

File: server/views/novels-main.pug (hunk 6 above)
Comment (AleksandraZhi) [reply server/views/novels-main.pug]: This is cleaned now.

File: server/models/Books.js (hunk 3 above)
Comment (mforner13) [reply server/models/Books.js:8]: Ahh nice! I would remove the comment on line 7 then, and it also seems like you wouldn't need the `bookType` field :) 

If they're all novels as well, you could just use `title`, `chapter`, `path` as the fields to simplify it - as the model's called `novel` so it's already clear it refers to a novel :)

Comment (AleksandraZhi) [reply server/models/Books.js:8]: Thank you, noted. This line is now removed.

**Comment by AleksandraZhi** [comment]
Thank you, I've removed this line now.
Aleksandra.

**Review by mforner13 — APPROVED** [review]
Well done looks great 😄

**Comment by AleksandraZhi** [comment]
Omg! That is amazing! Thank you!

**Pull request merged**